src/components/Composer.vue

```diff
@@ -83,6 +83,9 @@
 				@keyup="onInputChanged"
 			/>
 		</div>
+		<div v-if="noSubject" class="warning">
+			{{ t('mail', 'This mail does not yet have a subject.') }}
```

Comment:
```suggestion
			{{ t('mail', 'This mail does not have a subject yet.') }}
```

Not 100% sure which one is the correct one but this sounds a tad better